python/chronos/src/bigdl/chronos/autots/model/auto_lstm.py

```diff
@@ -90,10 +88,19 @@ def __init__(self,
         )
         self.metric = metric
         self.metric_mode = metric_mode
-        model_builder = PytorchModelBuilder(model_creator=model_creator,
-                                            optimizer_creator=optimizer,
-                                            loss_creator=loss,
-                                            )
+
+        if backend.startswith("torch"):
+            from bigdl.chronos.model.VanillaLSTM_pytorch import model_creator
+            model_builder = PytorchModelBuilder(model_creator=model_creator,
+                                                optimizer_creator=optimizer,
+                                                loss_creator=loss)
+        elif backend.startswith("keras"):
+            from bigdl.chronos.model.tf2.VanillaLSTM_keras import model_creator
+            model_builder = KerasModelBuilder(model_creator=model_creator,
+                                              optimizer=optimizer,
+                                              loss=loss)
```

Comment:
This automodel is inherited from BasePytorchAutomodel, may be we can create 2 automodels here called `AutoPytorchLSTM` and `AutoKerasLSTM`. `backend` parameters can be implemented under `AutoModelFactory`